.github/workflows/sysroot.yml

```diff
@@ -25,7 +29,7 @@ jobs:
           sudo apt-get update
           sudo apt-get install -y gcc g++ gperf bison flex texinfo help2man make libncurses5-dev \
           python3-dev autoconf automake libtool libtool-bin gawk wget bzip2 xz-utils unzip \
-          patch rsync meson ninja-build curl
+          patch rsync meson ninja-build curl lzip
```

Comment:
Why do you need `lzip`?

Reply:
apologies, lzip was required by my crosstools-ng config because is was available in the system when I built crosstools itself. 

since the crosstools version in the CI doesn't require lzip, it can be removed here. I have also removed the corresponding line from the s390x config in 62d21783b9e6c7412429c9402e99d90b52346276